scripts/moveFiles.bash

```diff
@@ -6,6 +6,10 @@ pushd ../publicgoods-website/registry && \
     git add static/* && \
     cp ../../publicgoods-scripts/packages/registry/build/index.html . && \
 popd
+pushd ../publicgoods-website/eligibility && \
+    cp -R ../../publicgoods-website/wp-content/themes/dpga/font/. ../../publicgoods-scripts/packages/eligibility/src/font/ && \
```

Comment:
I claim that instead of copying you can create the two symlinks that will cover a wider range of cases, and I argue that the error appears in the build of `packages/eligibility` so the following softlinks will solve it (replace lines 9 thru 12 with the following):
```bash
pushd packages/eligibility && \
  ln -s ../../../../publicgoods-website/wp-includes/
  ln -s ../../../../publicgoods-website/wp-content/
  npm install && npm run build && \
popd

Reply:
I am new to using `ln` however, tested it but the build failed [here](https://github.com/nathanfletcher/publicgoods-scripts/runs/3125146843?check_suite_focus=true#step:11:58). It could not find the fonts.

Reply:
No worries. I discovered what the error is.
I altered the font symlinks for the purpose of my solution. I'm changing them back to the original links and testing again.